src/plugins/orangehrmRecruitmentPlugin/config/routes.yaml

```
@@ -6,14 +11,15 @@ candidate_schedule_interview:
 candidate_schedule_interview_history:
   path: /recruitment/jobInterview/history/{hid}/interviewer/{id}
   controller: OrangeHRM\Recruitment\Controller\ScheduleInterviewHistory::handle
+  methods: [ GET ]
 
 candidate_shortlist:
   path: /recruitment/vacancy/{vid}/action/{aid}
   controller: OrangeHRM\Recruitment\Controller\ShortlistCandidateController::handle
   methods: [ GET ]
 
 view_shortlist_history:
-  path: /recruitment/VacancyStatus
+  path: /recruitment/VacancyStatus/{hid}
```

```suggestion
  path: /recruitment/vacancyStatus/{historyId}
```
better to follow camelCase